src/corona_hakab_model/supervisor.py

```diff
@@ -14,6 +14,7 @@
 
 import numpy as np
 import pandas as pd
+import os
```

Comment:
i dont think os is being used in this file anymore

Reply:
it is...